app/views/songs/index.html.erb

```diff
@@ -1,29 +1,47 @@
-<h1>Songs</h1>
-<table class="songs-table">
-  <thead>
-    <tr>
-      <th>Name</th>
-      <th>Artist</th>
-      <th>Tempo</th>
-      <th>Key</th>
-    </tr>
-  </thead>
-</table>
 
+<%= stylesheet_link_tag 'songs_manifest' %>
+
+<div class="index-page">
+  <div class="container">
+    <div class="row">
+      <div class="center-block">
+        <h1>Songs</h1>
+        <input type="search" id="songs-search-field" class="form-control" placeholder="Enter a lyric phrase, artist, or song title"/>
```

Comment:
We're actually not searching the artist field right now, so can you take that out?